src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -135,6 +136,42 @@ internal ReflectTypeDescriptionProvider()
             [s_intrinsicReferenceKey] = typeof(ReferenceConverter),
         });
 
+        /// <summary>
+        /// This is a table we create for parameterless constructor funcs for intrinsic-type converters.
+        /// Converters that take Types in their constructors are not cached.
+        /// </summary>
+        private static Dictionary<Type, Func<object>> ConverterConstructorFuncs => LazyInitializer.EnsureInitialized(ref s_converterConstructorFuncs, () => new Dictionary<Type, Func<object>>()
```

Comment:
Any reason not to just initialize it right away?

Reply:
I have it consistent with the rest of the caches in this file. Perhaps @safern or @eerhardt have some thoughts.